napari_aicssegmentation/_dock_widget.py

```diff
@@ -166,6 +169,30 @@ def set_step_3_layout(self, enabled=False):
             if enabled is False:
                 button.setDisabled(True)
             self.page.layout().addWidget(button, alignment=Qt.AlignCenter)
+    
+    """ 
+    Add CollapsibleBox widgets in 3 different initial states (enabled and closed, enabled
+    and open, and disabled) to the bottom of the window as examples
+    """
+    def add_collapsible_boxes(self):
+        layout_1 = QVBoxLayout()
+        layout_1.addWidget(QLabel("This is a label"))
+        layout_1.addWidget(QPushButton("This is a button"))
+        self.page.layout().addWidget(
+            CollapsibleBox("<span>1.&nbsp;Intensity normalization</span>", layout_1, isOpen=False)
+        )
+
+        layout_2 = QVBoxLayout()
+        layout_2.addWidget(QLabel("This is a label"))
+        layout_2.addWidget(QPushButton("This is a button"))
+        self.page.layout().addWidget(CollapsibleBox("<span>2.&nbsp;Smoothing</span>", layout_2))
+
+        layout_3 = QVBoxLayout()
+        layout_3.addWidget(QLabel("This is a label"))
+        layout_3.addWidget(QPushButton("This is a button"))
+        box_3 = CollapsibleBox("<span>3.&nbsp;2D filament filter</span>", layout_3, isOpen=False)
+        box_3.setDisabled(True)
```

Comment:
@Aerendel This is how you disable the collapsible box widget

Reply:
Nice, I guess that means QT must look for child widgets to disable as well - I wasn't sure that would be the case.